package.json

```diff
@@ -25,13 +25,14 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "react": ">=15.4.1",
-    "react-dom": ">=15.4.1"
+    "react": ">=15.5.4",
```

Comment:
Is this really the minimum peer version? With `create-react-class` and `prop-types` you should be compatible back to 15.0 or so.